release/models/platform/openconfig-platform.yang

```diff
@@ -531,6 +539,36 @@ module openconfig-platform {
         this reports the role of the component.";
     }
 
+    container last-poweroff-reason {
+      description
+        "Records last power-off reason for a component.";
+
+      uses oc-platform-types:component-last-power-off-reason-trigger;
+    }
+
+    leaf last-poweroff-time {
+      type oc-types:timeticks64;
+      units "nanoseconds";
+      description
+        "This reports the time of the last shutdown of the component. 
+         The value is the timestamp in nanoseconds relative to the Unix Epoch (Jan 1, 1970 00:00:00 UTC). 
+         If a system is rebooted and after the configuration is loaded, say the controller-card is identified 
+         to be shutdown in configuration, the card should be powered-off and 'last-power-off-time' 
+         leaf should be recorded.";
+    }
+
+    leaf last-switchover-time {
+      type oc-types:timeticks64;
+      units "nanoseconds";
+      description
+        "For components that have redundant roles (e.g. two
+        supervisors in a device, one as primary the other as
+        secondary), this reports the time of the last change of
+        the component's role. The value is the timestamp in
+        nanoseconds relative to the Unix Epoch (Jan 1, 1970 00:00:00 UTC).";
+
+    }
```

Comment:
This should be removed.  It looks like a cut-paste error?  It is already defined at https://github.com/openconfig/public/pull/1041/files#diff-f787893a2471f7ec1785fa010e4981f339dc68a144cffb802df9afcdf02a30b5R582

Reply:
Friendly reminder for this

Reply:
done!